specs/cloud-defend.spec.yml

```diff
@@ -0,0 +1,24 @@
+version: 2
+inputs:
+  - name: cloud_defend
+    description: "Cloud defend for containers"
+    platforms: &platforms
+      - container/amd64
+      - container/arm64
+    outputs: &outputs
+      - elasticsearch
+    command:
+      args: &args
+        - "-E"

```

Comment:
oh these will cause our program to fail. We aren't a beat, we're more standalone like endpoint.

Reply:
I see. Ok, i'll remove.